packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -562,34 +626,109 @@ class RenderListWheelViewport
       perspective: _perspective,
     );
 
-    context.pushTransform(
-      // Text with TransformLayers and no cullRects currently have an issue rendering
-      // https://github.com/flutter/flutter/issues/14224.
-      false,
-      offset,
-      _centerOriginTransform(transform),
-      // Pre-transform painting function.
-      (PaintingContext context, Offset offset) {
-        context.paintChild(
-          child,
-          offset + new Offset(
-            untransformedPaintingCoordinates.dx,
-            // Paint everything in the center (e.g. angle = 0), then transform.
-            -_topScrollMarginExtent,
-          ),
-        );
-      },
-    );
+    void _paintCylindrically(PaintingContext context, Offset offset) {
+      context.pushTransform(
+        // Text with TransformLayers and no cullRects currently have an issue rendering
+        // https://github.com/flutter/flutter/issues/14224.
+        false,
```

Comment:
I haven't tested this in a while. Does it still work if this false was needsCompositing instead?

Reply:
I have tried and it still leads to rendering error.